### PR TITLE
Add Actions to GitHub for CI/CD

### PR DESCRIPTION
**Pull request by dgtlrift**
There are some gaps with Makefile generation calling out variables for SVNTIME expanding to dieharder.svn.time which may need to be refactored to equivalent operations in Git - ie svn2cl converted to use (and install) git2cl

### CODE REVIEW

File: .github/workflows/c-cpp.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [ "actions-test" ]
+  pull_request:
+    branches: [ "actions-test" ]
```
Comment (eddelbuettel) [inline .github/workflows/c-cpp.yml]: Those branches would need changing.

**Comment by eddelbuettel** [comment]
I usually go with an essentially empty

```yaml
on:
  push:
  pull_request:
```

**Review by eddelbuettel — APPROVED** [review]
:+1:

File: .github/workflows/c-cpp.yml (hunk 1 above)
Comment (dgtlrift) [reply .github/workflows/c-cpp.yml]: Updated per comment below with 
```
on:
  push:
  pull_request:
```

**Pull request merged**